scripts/run.power-mpc5777m-evb.sh

```diff
@@ -0,0 +1,93 @@
+#This script is made to be sourced by the run.sh script. Do not use it alone!
```

Comment:
Okay, this script probably does not work anymore, but okay. We will wait for the framework to witness its rebirth.